hw_4/sql_flask_hw4.py

```diff
@@ -0,0 +1,64 @@
+from webargs import validate, fields
+from webargs.flaskparser import use_kwargs
+
+from flask import Flask, render_template
+
+from sql_handler import execute_query
+
+app = Flask(__name__)
+
+
+# choice page
+@app.route('/')
+def index():
+    return '<h3>Start page (tap choice): <a href = "country_sales">country_sales</a> (?country=..), ' \
+           '<a href = "track_id"> track_id</a> (?id=..), <a href = "time_all_tracks"> time_all_tracks</a>.</h3>'
+
+
+# country sales
+@app.route('/country_sales')
+@use_kwargs({"country": fields.Str(load_default="%"), }, location="query")
+def get_country_sales(country):
+    query = f"SELECT UnitPrice AS Price_Unit, count(Quantity) AS Total_Quanity,(count(Quantity) * UnitPrice) AS Total_Sales " \
+            f"FROM invoice_items as i1 " \
```

Comment:
you need to group by country if you want to use this query
and with group by use having command

Reply:
Ok.Correct

Reply:
All correcting